.github/workflows/datamodel-test.yml

```
@@ -39,7 +39,7 @@ jobs:
       - name: run tests
         run:  docker exec teksi-wastewater pytest
 
-      - name: failure logs
+      - name: logs
```

```suggestion
      - name: failure logs
```
Keep failure logs as they are only displayed if failure happens